docker-compose.yml

```diff
@@ -94,6 +96,8 @@ services:
     stop_grace_period: "3s"
     volumes:
       - "pgdata:/var/lib/postgresql/data"
+    ports:
+      - "54322:5432"
```

Comment:
You can probably even open this up on the default port

Reply:
I cannot, as the `db-test` opens the default port. Also it kind of clashes of course with any native postgres dbs.

I could change the `db-test` to an alternative port, change this to default and be specific about, that you cannot run another postgres db on default port, when launching standard compose. You have then to alter either your compose service, or your running db to make it work.

What do you think?